README.rst

```diff
@@ -7,17 +7,65 @@ edx-bulk-grades
 Support for bulk scoring and grading. This adds models and an API for reading and modifying
 scores and grades in bulk.
 
-Overview (please modify)
-------------------------
+Overview
+---------
 
 The ``README.rst`` file should then provide an overview of the code in this
 repository, including the main components and useful entry points for starting
 to understand the code in more detail.
+edx-bulk-grades is a library that runs under lms. It uses the configuration settings defined in lms as well.
+In order to use, the library must be installed into edx-platform.
 
-Documentation
--------------
+Using with Docker Devstack
+--------------------------
+Prerequisite: Have your Open edX https://github.com/edx/devstack properly installed.
+Note: When you see "from inside the lms" below, it means that you've run ``make lms-shell`` from your devstack directory
+and are on a command prompt inside the LMS container.
 
-The full documentation is at https://edx-bulk-grades.readthedocs.org.
+1. Clone this repo into ../src/ directory (relative to your "devstack" repo location). This will mount the directory in
+a way that is accessible to the lms container.
+
+2. From inside the lms, uninstall bulk-grades and reinstall your local copy. You can just copy the following line::
+
+    pip uninstall edx-bulk-grades -y; pip install -e /edx/src/edx-bulk-grades
+
+3. Now, get your bulk-grades development environment set up::
+
+    cd /edx/src/edx-bulk-grades
+    virtualenv edx-bulk-grades-env
+    source edx-bulk-grades-env/bin/activate
+    make requirements
+
+Making Code Changes
+-------------------
+1. After checking out a new branch, increment __version__ by the smallest possible value located in
```

Comment:
Nit: it's helpful to wrap "\_\_version\_\_" in double back-ticks, i.e. \`\`\_\_version\_\_\`\`